src/views/Stopwatch.vue

```diff
@@ -10,7 +10,7 @@ div
   b-input-group(size="lg")
     b-input(v-model="label" placeholder="What are you working on?")
     b-input-group-append
-      b-button(@click="startTimer(label)", variant="success")
+      b-button(@click="startTimer(label, '')", variant="success")
```

Comment:
Shouldn't you pass in the tags here? I also don't see the `b-input` for the tags.